src/library/Connect/WorkloadInformationProvider.cs

```diff
@@ -117,6 +122,26 @@ internal class WorkloadInformationProvider : IWorkloadInformationProvider
             }
         }
 
+        public List<int> GetPortsToIgnoreFromAnnotations(V1ServiceList serviceNameSpace) {
+            var portsToIgnore = new List<int>();
+            List<V1Service> portsList = serviceNameSpace.Items.Where(item => item.Metadata?.Annotations?.ContainsKey(ServiceAnnotations) ?? false).ToList();
+            portsList.ForEach(port => {

```

Comment:
once you rename the variable this should be "forEach(service => {" instead of "ForEach(port =>"